app/src/main/java/com/vinh/dictionary_1/screen/splash/SplashPresenter.java

```diff
@@ -1,10 +1,13 @@
 package com.vinh.dictionary_1.screen.splash;
 
+import android.app.Activity;
 import android.content.Context;
 import com.vinh.dictionary_1.MainApplication;
 import com.vinh.dictionary_1.R;
 import com.vinh.dictionary_1.data.source.local.sharedpref.SharedPrefsImplement;
+import com.vinh.dictionary_1.screen.home.HomeActivity;
```

Comment:
ERROR: Unused import: com.vinh.dictionary_1.screen.home.HomeActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports